src/Adapter/Customer/CommandHandler/BulkDeleteCustomerHandler.php

```diff
@@ -56,5 +58,17 @@ public function handle(BulkDeleteCustomerCommand $command)
             $customer->deleted = true;
             $customer->update();
         }
+
+        PrestaShopLogger::addLog(
+            Context::getContext()->getTranslator()->trans(
+                'Customers bulk deleted',
+                [],
+                'Admin.Advparameters.Notification'
+            ),
+            1,
+            null,
+            'Customer',
+            $customer->id
```

Comment:
The variable `$customer`is not available in this context, maybe put this function call in the loop so we can keep track of all customer ids that have been deleted?

Reply:
done